speech/google/cloud/gapic/speech/v1/speech_client_config.json

```diff
@@ -6,9 +6,7 @@
           "DEADLINE_EXCEEDED",
           "UNAVAILABLE"
         ],
-        "non_idempotent": [
-          "UNAVAILABLE"
-        ]
+        "non_idempotent": []
```

Comment:
This bit is intentional?  Most of the others of these I have seen allow retries for `UNAVAILABLE`, even for non-idempotent methods.